docs/configuration.rst

```diff
@@ -25,11 +25,18 @@ recreating it each time you run the test. This flag enables
 ``--runner-class``
 ******************
 
-Full Python dotted path to a package, module, Django TestRunner. 
+Full Python dotted path to the `Django test runner`_ module used for your
+BDD test suite.  Default: |BehaviorDrivenTestRunner|_
 
-Not to be confused with `--behave-runner-class` that handles the internal
-`TestRunner` inside `behave`. You can read more about it in the
-`behave docs <https://behave.readthedocs.io/en/latest/behave.html#cmdoption-runner-class>`__.
+You can use this option if you require custom behavior that deviates from
+`Django's default test runner`_ and is hard or impossible to configure.
+
+.. note::
+
+    Not to be confused with ``--behave-runner-class`` that handles the
+    internal `test runner inside behave`_.  You would use that to override
+    *behave*'s feature/step file discovery and similar behavior.  Read more
+    about it in the |behave docs (runner class)|_.
```

Comment:
@kingbuzzman Are we sure that combining the CLI options `--behave-runner-class` and `--runner-class` will work?

Also, might it be more explicit to call the option `--django-runner-class`, maybe? :thinking:  The current solution is concise(r), but with "django" there would be little chance for confusion.

Reply:
> @kingbuzzman Are we sure that combining the CLI options --behave-runner-class and --runner-class will work?

No, they're fundamentally different, and cannot be interchanged/combined.

> Also, might it be more explicit to call the option --django-runner-class, maybe? 🤔 The current solution is concise(r), but with "django" there would be little chance for confusion.

The issue with this is, how it reads: 
`python manage.py behave --runner-class SomeClassRunner`
vs
`python manage.py behave --django-runner-class SomeClassRunner` 

This may be just me, but when i see `manage.py` I'm already thinking Django, thus adding `django` prefix, to me, seems redundant.

Reply:
Agreed. The `manage.py` makes the difference.

As for combining, you could use two custom runners, right? One for *behave* and one for Django, and use them together, e.g.

```console
python manage.py behave --runner-class MyDjangoTestRunner --behave-runner-class MyBehaveRunner
```

That *can* be combined, can't it? – Will it work?

Reply:
Sorry, misunderstood your original meaning; yes! You can have both in the same command